src/simple-length.js

```diff
@@ -67,6 +67,25 @@
     return new SimpleLength((this.value - subtractedLength.value), this.type);
   };
 
+  SimpleLength.prototype._convertToCalcLength = function() {
+    var calcDictionary = {};
+
+    //Set calcDictionary field to the Length type inside the SimpleLength object that called the function
+    calcDictionary[this.type] = this.value;
+
+    return new CalcLength(calcDictionary);
+  }
+
+  SimpleLength.prototype.equals = function(lengthValue) {
+    if (!(lengthValue instanceof SimpleLength)){
+      return false;
+    }
+    if (!(this.type == lengthValue.type && this.value == lengthValue.value)){
```

Comment:
It's shorter and clearer to write

if (this.type != lengthValue.type || this.value != lengthValue.value)